lib/pka.c

```diff
@@ -40,6 +40,8 @@ struct poll_args {
     ecc_point_t   **ecc_point;
 };
 
+static pka_atomic32_t pka_async_curr_jobs_cnt;

```

Comment:
Question: do you know if such implementation has an impact on the current performance?

Reply:
For synchronous, it’s not noticeable. With asynchronous and high performance numbers, the impact is ~ 5%

Reply:
Thanks Shih-Yi.